### PR TITLE
Fix duplicate variable name error

### LINKED ISSUE #1199: Redeclared symbol error when APIs with basic auth secured endpoints
### Steps to reproduce:
init a project
Copy 2 API definitions which has production/sandbox endpoints secured with basic auth to the project.
build the project.

Error:
```
Compiling source
	wso2/dev:3.1.0
error: wso2/dev:3.1.0::Basic_Auth__2_0_0.bal:12:32: redeclared symbol 'prod_OutboundBasicAuthProvider'
error: wso2/dev:3.1.0::Basic_Auth__2_0_0.bal:17:23: redeclared symbol '_prodOutboundBasicAuthHandler'
error: wso2/dev:3.1.0::Basic_Auth__2_0_0.bal:41:32: redeclared symbol 'sand_OutboundBasicAuthProvider'
error: wso2/dev:3.1.0::Basic_Auth__2_0_0.bal:46:23: redeclared symbol '_sandOutboundBasicAuthHandler'
```

caused by lines - variable naming in https://github.com/wso2/product-microgateway/blob/60288b120106d07dec210c170ca7a8b02791ad65/components/micro-gateway-cli/src/main/resources/templates/basicAuth.mustache#L3, https://github.com/wso2/product-microgateway/blob/60288b120106d07dec210c170ca7a8b02791ad65/components/micro-gateway-cli/src/main/resources/templates/basicAuthHandler.mustache#L2
### Affected Product Version:

3.1.0 

### Environment details (with versions):
- OS:
- Client:
- Env (Docker/K8s):

---

### PR DESCRIPTION
**Pull request by AmaliMatharaarachchi**
### Purpose

Rename variable naming in basic auth template

### Issues

Fixes #1199

### Automation tests
 - Unit tests added: No
 - Integration tests added: No

### Tested environments

catalina
---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: components/micro-gateway-cli/src/main/resources/templates/basicAuth.mustache [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {{#securityConfig}}{{#equals securityConfig.type "basic"}},
     auth: {
-        authHandler: {{operationId}}_{{endpointUrlType}}OutboundBasicAuthHandler
+        authHandler: {{qualifiedServiceName}}{{operationId}}_{{endpointUrlType}}OutboundBasicAuthHandler
     }{{/equals}}{{/securityConfig}}
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/resources/templates/basicAuth.mustache:4]:
```suggestion
    }{{/equals}}{{/securityConfig}}

```

File: components/micro-gateway-cli/src/main/resources/templates/basicAuthHandler.mustache [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {{#securityConfig}}{{#equals securityConfig.type "basic"}}
-auth:OutboundBasicAuthProvider {{endpointUrlType}}_{{operationId}}OutboundBasicAuthProvider = new({
+auth:OutboundBasicAuthProvider {{endpointUrlType}}_{{qualifiedServiceName}}{{operationId}}OutboundBasicAuthProvider = new({
     username: gateway:retrieveConfig("{{name}}_{{endpointUrlType}}_basic_username", "{{securityConfig.username}}"),
     password: gateway:retrieveConfig("{{name}}_{{endpointUrlType}}_basic_password", "")
 });
 
-http:BasicAuthHandler {{operationId}}_{{endpointUrlType}}OutboundBasicAuthHandler = new({{endpointUrlType}}_{{operationId}}OutboundBasicAuthProvider);
+http:BasicAuthHandler {{qualifiedServiceName}}{{operationId}}_{{endpointUrlType}}OutboundBasicAuthHandler = new({{endpointUrlType}}_{{qualifiedServiceName}}{{operationId}}OutboundBasicAuthProvider);
 {{/equals}}{{/securityConfig}}
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/resources/templates/basicAuthHandler.mustache:8]:
```suggestion
{{/equals}}{{/securityConfig}}

```

File: components/micro-gateway-core/src/main/ballerina/Ballerina.toml [hunk 3]
```diff
@@ -6,7 +6,7 @@ version= "3.1.0"
 target = "java8"
 
     [[platform.libraries]]
-    path = "../../../target/org.wso2.micro.gateway.core-3.1.0.jar"
+    path = "../../../target/org.wso2.micro.gateway.core-3.1.1-SNAPSHOT.jar"
```
Comment (praminda) [inline components/micro-gateway-core/src/main/ballerina/Ballerina.toml:9]: We need to update master version number to 3.2.0 or next major version 4.0. Then these values may change. So hold on 😄

File: components/micro-gateway-cli/src/main/resources/templates/basicAuth.mustache (hunk 1 above)
Comment (AmaliMatharaarachchi) [reply components/micro-gateway-cli/src/main/resources/templates/basicAuth.mustache]: Fixed

File: components/micro-gateway-cli/src/main/resources/templates/basicAuthHandler.mustache (hunk 2 above)
Comment (AmaliMatharaarachchi) [reply components/micro-gateway-cli/src/main/resources/templates/basicAuthHandler.mustache]: Fixed in https://github.com/wso2/product-microgateway/pull/1200/commits/b34bf78c712b9c1122f77b28a6fbe3f26f1d37bc

**Pull request merged**